src/main/java/teammates/common/datatransfer/attributes/FeedbackQuestionAttributes.java

```diff
@@ -159,30 +159,27 @@ public String getJsonString() {
     public List<String> getInvalidityInfo() {
         FieldValidator validator = new FieldValidator();
         List<String> errors = new ArrayList<String>();
-        String error;
 
-        error = validator.getInvalidityInfoForFeedbackSessionName(feedbackSessionName);
-        if (!error.isEmpty()) {
-            errors.add(error);
-        }
+        errors.add(validator.getInvalidityInfoForFeedbackSessionName(feedbackSessionName));
 
-        error = validator.getInvalidityInfoForCourseId(courseId);
-        if (!error.isEmpty()) {
-            errors.add(error);
-        }
+        errors.add(validator.getInvalidityInfoForCourseId(courseId));
 
-        error = validator.getInvalidityInfoForEmail(creatorEmail);
+        // special case when additional text should be added to error text
+        String error = validator.getInvalidityInfoForEmail(creatorEmail);
         if (!error.isEmpty()) {
-            errors.add("Invalid creator's email: " + error);
+            StringBuffer stringbuff = new StringBuffer();
```

Comment:
Thanks! Now found this section in Supplementary documents, just didn't reach this section previously